src/pages/cadastro.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+<head>
+    <meta charset="UTF-8">
+    <meta name="author" content="diogo">
+    <link rel="stylesheet" href="../styles/login.css" />
+    <title>Cadastro</title>
+</head>
+<body>
+    <div id="main-container-cadastro">
+        <h1>Cadastre-se</h1>
+    
+        <form id="register-form">
+        <div class="full-box">
+            <label for="text">Nome</label>
+            <input type="text" name="name" id="name" placeholder="Digite seu nome" />
+        </div>
+        <div class="full-box">
+            <label for="text">Sobrenome</label>
```

Comment:
Corrigir o valor do atributo `for` do elemento `label` para que contenha o valor do atributo `name` do input ao qual pertence.